pyanaconda/ui/gui/spokes/storage.glade

```diff
@@ -214,6 +214,9 @@
       <action-widget response="0">need_space_cancel_button</action-widget>
       <action-widget response="3">need_space_reclaim_button</action-widget>
     </action-widgets>
+    <child>
+      <placeholder/>
+    </child>
```

Comment:
I think that you can remove this placeholder.

Reply:
I'd rather keep it in so that
```
-<!-- Generated with glade 3.20.0 -->
+<!-- Generated with glade 3.20.3 -->
```
is true.

Reply:
Ok.